server/README.md

```diff
@@ -0,0 +1,91 @@
+# NextRound: Server
+
+## Routes
+
+### GET
+
+#### /cycle
+
+provides all the information about the current users current cycle
+
+#### /cycle/:cycleId
+
+provides all the information about the current users cycle with given `cycleId`
+
+#### /application/:applicationId
+
+provides all the information about the current user's application from the current cycle with given `applicationId`
+
+#### /application/:cycleId/:applicationId
+
+provides all the information about the current user's application from the cycle  with given `cycleId` with given `applicationId`
+
+### POST
+
+#### /signin
+
+handle storing a session cookie and working with firebase auth as an admin
+
+#### /signup
+
+might be the same as login depending on how firebase works? We'll see when we start coding it
+
+#### /logout
+
+telling firebase you are logging out and to remove authtoken from request headers
+
+#### /cycle
+
+makes a new cycle for the current user
+
+#### /application
+
+make a new application in the current cycle for the current user
+
+#### /application/event/:applicationId

```

Comment:
Is this supposed to have a cycleID since the route is repeated right under this for the current cycle?

Reply:
Well, actually we don't want to be able to add to previous cycles, so remove this